ebpf/cgroup_linux.go

```diff
@@ -0,0 +1,23 @@
+//go:build linux
+
+package ebpf
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+)
+
+const mountPoint = "/mnt/cgroup2"
+
+func mountCgroup2() error {
+	err := os.Mkdir(mountPoint, 0755)
+	if err != nil {
+		return fmt.Errorf("creating directory at %q: %w", mountPoint, err)
+	}
+	err = syscall.Mount("none", mountPoint, "cgroup2", 0, "")
```

Comment:
maybe just add some comments for a context why we need this? + this 
 link https://docs.kernel.org/admin-guide/cgroup-v2.html#mounting ? 🤔